server/repository/src/migrations/v2_05_00/add_email_queue_table.rs

```diff
@@ -0,0 +1,52 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_email_queue_table"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        if cfg!(feature = "postgres") {
+            sql!(
+                connection,
+                r#"
+                CREATE TYPE email_queue_status_enum AS ENUM (
+                    'QUEUED',
+                    'SENT',
+                    'ERRORED',
+                    'FAILED',
```

Comment:
```suggestion
                    'FAILED'
```
I don't think SQL likes a trailing `comma` :)

Reply:
Added here 56690bedfddd39b76e00ca771067e77ae9d102a6 :)